File: TCDNew-master/src/ui.android/lib/models/dtos/PurchaseTaskLineReadByBarcodeReq.dart
class PurchaseTaskLineReadByBarcodeReq {
  String purchaseTaskId;
  String barcode;
  String currentPalletCode;

  PurchaseTaskLineReadByBarcodeReq(this.purchaseTaskId, this.barcode, this.currentPalletCode);

  toJson() {
    return {
      'purchaseTaskId': purchaseTaskId,
      'barcode': barcode,
      'currentPalletCode': currentPalletCode,
    };
  }
}

File: TCDNew-master/src/api/Cen.Wms.Domain.Purchase.Api/Dtos/PurchaseTaskLineReadByBarcodeReq.cs
using System;

namespace Cen.Wms.Domain.Purchase.Api.Dtos
{
    public class PurchaseTaskLineReadByBarcodeReq
    {
        public Guid PurchaseTaskId { get; set; }
        public string Barcode { get; set; }
        public string CurrentPalletCode { get; set; }
    }
}